fdbkubernetesmonitor/main.go

```diff
@@ -47,6 +47,8 @@ var (
 	mainContainerVersion string
 	additionalEnvFile    string
 	listenAddress        string
+	certFile             string
+	keyFile              string
```

Comment:
Those can be added below (line 86), I don't think those have to be global variables in this package?

Reply:
sorry just saw other args there and threw it in, oops!